src/coreclr/jit/emitriscv64.cpp

```diff
@@ -3360,6 +3402,48 @@ void emitter::emitDisInsName(code_t code, const BYTE* addr, instrDesc* id)
         }
         case 0x73:
         {
+            unsigned int opcode2 = (code >> 12) & 0x7;
+            const char* rd = RegNames[(code >> 7) & 0x1f];
+            int csrtype = (code >> 20);
+            if (opcode2 <= 0x3)
+            {
+                const char*  rs1     = RegNames[(code >> 15) & 0x1f];
+                switch (opcode2)
+                {
+                    case 0x1: // CSRRW
+                        printf("csrrw           %s, %d, %s\n", rd, csrtype, rs1);
+                        return;
+                    case 0x2: // CSRRS
+                        printf("csrrs           %s, %d, %s\n", rd, csrtype, rs1);
+                        return;
+                    case 0x3: // CSRRC
+                        printf("csrrc           %s, %d, %s\n", rd, csrtype, rs1);
+                        return;
+                    default:
+                        printf("RISCV64 illegal instruction: 0x%08X\n", code);
+                        break;
+                }
+            }
+            else
+            {
+                int imm5 = ((code >> 15) & 0x1f);
```

Comment:
Nit: this value is `the five-bit zero-extended immediate`. I think unsigned int is better.

Reply:
Done.